test/run/ch-run_misc.bats

```diff
@@ -155,169 +155,169 @@ EOF
     echo "$output"
     [[ $status -eq 64 ]]
     [[ $output =~ 'option requires an argument' ]]
 
     # empty argument to --bind
-    run ch-run -b '' "$CHTEST_IMG" -- true
+    run ch-run -b '' "$ch_imgtest" -- true
     echo "$output"
     [[ $status -eq 1 ]]
     [[ $output =~ '--bind: no source provided' ]]
 
     # source not provided
-    run ch-run -b :/mnt/9 "$CHTEST_IMG" -- true
+    run ch-run -b :/mnt/9 "$ch_imgtest" -- true
     echo "$output"
     [[ $status -eq 1 ]]
     [[ $output =~ '--bind: no source provided' ]]
 
     # destination not provided
-    run ch-run -b "$IMGDIR/bind1:" "$CHTEST_IMG" -- true
+    run ch-run -b "$ch_imgdir/bind1:" "$ch_imgtest" -- true
     echo "$output"
     [[ $status -eq 1 ]]
     [[ $output =~ '--bind: no destination provided' ]]
 
     # source does not exist
-    run ch-run -b "$IMGDIR/hoops" "$CHTEST_IMG" -- true
+    run ch-run -b "$ch_imgdir/hoops" "$ch_imgtest" -- true
     echo "$output"
     [[ $status -eq 1 ]]
-    r="can't bind .+/hoops to $CHTEST_IMG/mnt/0: No such file or directory"
+    r="can't bind .+/hoops to $ch_imgtest/mnt/0: No such file or directory"
     [[ $output =~ $r ]]
 
     # destination does not exist
-    run ch-run -b "$IMGDIR/bind1:/goops" "$CHTEST_IMG" -- true
+    run ch-run -b "$ch_imgdir/bind1:/goops" "$ch_imgtest" -- true
     echo "$output"
     [[ $status -eq 1 ]]
-    r="can't bind .+/bind1 to $CHTEST_IMG/goops: No such file or directory"
+    r="can't bind .+/bind1 to $ch_imgtest/goops: No such file or directory"
     [[ $output =~ $r ]]
 
     # neither source nor destination exist
-    run ch-run -b "$IMGDIR/hoops:/goops" "$CHTEST_IMG" -- true
+    run ch-run -b "$ch_imgdir/hoops:/goops" "$ch_imgtest" -- true
     echo "$output"
     [[ $status -eq 1 ]]
-    r="can't bind .+/hoops to $CHTEST_IMG/goops: No such file or directory"
+    r="can't bind .+/hoops to $ch_imgtest/goops: No such file or directory"
     [[ $output =~ $r ]]
 
     # correct bind followed by source does not exist
-    run ch-run -b "$IMGDIR/bind1:/mnt/9" -b "$IMGDIR/hoops" "$CHTEST_IMG" -- true
+    run ch-run -b "$ch_imgdir/bind1:/mnt/9" -b "$ch_imgdir/hoops" "$ch_imgtest" -- true
     echo "$output"
     [[ $status -eq 1 ]]
-    r="can't bind .+/hoops to $CHTEST_IMG/mnt/1: No such file or directory"
+    r="can't bind .+/hoops to $ch_imgtest/mnt/1: No such file or directory"
     [[ $output =~ $r ]]
 
     # correct bind followed by destination does not exist
-    run ch-run -b "$IMGDIR/bind1" -b "$IMGDIR/bind2:/goops" "$CHTEST_IMG" -- true
+    run ch-run -b "$ch_imgdir/bind1" -b "$ch_imgdir/bind2:/goops" "$ch_imgtest" -- true
     echo "$output"
     [[ $status -eq 1 ]]
-    r="can't bind .+/bind2 to $CHTEST_IMG/goops: No such file or directory"
+    r="can't bind .+/bind2 to $ch_imgtest/goops: No such file or directory"
     [[ $output =~ $r ]]
 }
 
 @test 'broken image errors' {
     scope standard
-    IMG="$BATS_TMPDIR/broken-image"
+    img="$BATS_TMPDIR/broken-image"
```

Comment:
If we use `img` for the one in `common.bash`, this should be something different to not shadow it — maybe?